[arrow] update

atkawa7 opened the pull request.

KindDragon committed:
[leptonica] Initial port

KindDragon committed:
[tesseract] Initial port. Closes #465

UnaNancyOwen committed:
Add WinPCAP Developer's Pack

Add WinPCAP Developer's Pack

UnaNancyOwen committed:
Build WinPCAP libraries from source code

Build WinPCAP libraries from source code.

UnaNancyOwen committed:
Add support to change linkage of CRT and library

Add support to change linkage of CRT and library.

mmha committed:
[KF5] Initial port of KPlotting

FabienPean committed:
Add CMake function vcpkg_from_bitbucket

FabienPean committed:
[blaze] Use vcpkg_from_bitbucket in portfile

paercebal committed:
Updated the documentation to reflect the greater tolerance on configs

Commit:
[openblas] Add patch for whitespace

thierryba committed:
protobuf: switching to 3.4

juandent committed:
This is my proposed new version of the port file

the port file for cppwinrt, latest release

atkawa7 committed:
[libstemmer] init

atkawa7 committed:
[libstemmer] add description

atkawa7 committed:
[gumbo] html parser init

atkawa7 committed:
[libunibreak] init port

atkawa7 committed:
[gumbo] fix missing header

atkawa7 committed:
[tinythread] init

atkawa7 committed:
[libmicrohttpd] init port

atkawa7 committed:
[picosha2] init

atkawa7 committed:
[fadbad] init port

atkawa7 committed:
[meschach] init

atkawa7 committed:
[lodepng] init port

atkawa7 committed:
[piex] init port

atkawa7 committed:
[bigint] init port

atkawa7 committed:
[butteraugli] init port

atkawa7 committed:
[guetzli] init fixes #844

Mixaill committed:
[tinyxml] add version 2.6.2

atkawa7 committed:
[paho-mqtt] update

Mixaill committed:
[vcpkg] allow underscores in package name

Mixaill committed:
[urdfdom_headers] add version 1.0.0

Mixaill committed:
[console_bridge] add version 0.3.2

Mixaill committed:
[urdfdom] add version 1.0.0

Mixaill committed:
[nlopt] add master 2017-08-24

atkawa7 committed:
[arrow] update

msftclas commented:
@atkawa7,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by Microsoft. We will now review your pull request.
_Thanks,
Microsoft Pull Request Bot_

thierryba committed:
code review

alexkaratarakis committed:
Merge pull request #1750 from thierryba/protobuf3.4

protobuf: switching to 3.4

alexkaratarakis committed:
Merge pull request #1753 from atkawa7/libstemmer

[libstemmer] init

alexkaratarakis committed:
Merge pull request #1754 from atkawa7/gumbo

[gumbo] init port

alexkaratarakis committed:
Merge pull request #1755 from atkawa7/libunibreak

[libunibreak] init port

alexkaratarakis committed:
Merge pull request #1756 from atkawa7/tinythread

[tinythread] init

alexkaratarakis committed:
Merge pull request #1757 from atkawa7/microhttpd

[libmicrohttpd] init port

alexkaratarakis committed:
Merge pull request #1758 from atkawa7/PicoSHA2

[picosha2] init

alexkaratarakis committed:
Merge pull request #1759 from atkawa7/fadbad

[fadbad] init port

alexkaratarakis committed:
Merge pull request #1760 from atkawa7/Meschach

[meschach] init

alexkaratarakis committed:
Merge pull request #1764 from atkawa7/lodepng

[lodepng] init port

AlexanderTaeschner committed:
Update to version 1.15.8.

Revert "Merge pull request #1277 from AlexanderTaeschner/CairoInitMutex".

This reverts commit 2aa83904eac9d11b69f09e9f0af85350fea472d9, reversing
changes made to 748e8b6e7495f06253eb21236bf3abf38b897b63.

alexkaratarakis committed:
Merge pull request #1765 from atkawa7/piex

[piex] init port

alexkaratarakis committed:
Merge pull request #1767 from atkawa7/bigint

[bigint] init port

alexkaratarakis committed:
Merge pull request #1768 from atkawa7/guetzli

[guetzli] init port

alexkaratarakis committed:
Merge pull request #1769 from Mixaill/tinyxml-262

[tinyxml] add version 2.6.2

jwillemsen committed:
Updated CONTROL file to list 6.4.4, portfile was updated some time ago already

    * ports/ace/CONTROL:

alexkaratarakis committed:
Merge pull request #1772 from atkawa7/paho-mqqt-update

[paho-mqtt] update

alexkaratarakis committed:
Merge pull request #1774 from Mixaill/nlopt

[nlopt] add master 2017-08-24

alexkaratarakis committed:
Merge pull request #1778 from jwillemsen/jwi-ace644fix

[ace] 6.4.4

alexkaratarakis committed:
Merge pull request #1777 from AlexanderTaeschner/Issue1289

[cairo] Update to version 1.15.8 and revert PR1277

alexkaratarakis committed:
Update CHANGELOG and bump version to v0.0.86

OlafvdSpek committed:
Update ctemplate, use Python 3

juandent committed:
These work correctly as far as I tested them...

pravic committed:
[meschach] Add version and description.

tobyc11 committed:
Update glslang shaderc spirv-tools
Update those ports to the versions listed in https://github.com/google/shaderc/blob/known-good/known_good.json

alexkaratarakis committed:
Merge pull request #1786 from pravic/patch-1

[meschach] Add version and description.

ras0219-msft committed:
Merge pull request #1657 from KindDragon/tesseract

Tesseract-OCR library

ShinNoNoir committed:
[qt5] Do not link all .lib files

This commit moves a few .lib files to subfolders to prevent them from
automatically getting linked:

Qt5Bootstrap (dbg):
This lib is linked against a release version of the CRT and is only
meant for release builds. For debug builds, this lib should not be
linked at all.

qtmain (rel), qtmaind (dbg),
Qt5AxServer (rel), Qt5AxServerd (dbg):
These libs are mutually exclusive. The user either links against
qtmain(d) for desktop applications or against Qt5AxServer(d) for ActiveX
servers.

See https://github.com/Microsoft/vcpkg/issues/1442 for more information.

ShinNoNoir committed:
[qt5] Bump version to reflect change in which .libs are linked

atkawa7 committed:
an empty delimiter causes the split function to fail

alexkaratarakis committed:
Merge pull request #1793 from atkawa7/patch-1

[vcpkg] an empty delimiter causes the split function to fail

Commit:
[openblas] Have the portfile apply the new patch

Commit:
[openblas] Bump version

alexkaratarakis committed:
Merge pull request #1749 from Alisayyy-Yang/master

[openblas] Fix build with white space in path

ras0219-msft committed:
Merge pull request #1790 from cyj0912/master

Update glslang shaderc spirv-tools

ras0219-msft committed:
Revert "[vcpkg] allow underscores in package name"

This reverts commit aef3577672b469ed915a7b0680d5e1ccf36347ed.

ras0219-msft committed:
[urdfdom-headers] [console-bridge] Rename to remove underscores

ras0219-msft committed:
[console-bridge] [urdfdom-headers] Move copyright location

alexkaratarakis committed:
[openblas] Use vcpkg_from_github()

jwillemsen committed:
[ACE] 6.4.5

alexkaratarakis committed:
Merge pull request #1794 from jwillemsen/jwi-ace-645

[ACE] 6.4.5

KindDragon committed:
[protobuf] Fix problem with define PROTOBUF_USE_DLLS

KindDragon committed:
[c-ares] Update version to 1.13.0

KindDragon committed:
[grpc] Update version to 1.6.0

codicodi committed:
[libuv] update to 1.14.1

alexkaratarakis committed:
[openblas] Move whitespace fixes to the whistespace.patch

alexkaratarakis committed:
Merge pull request #1796 from codicodi/libuv-1.14.1

[libuv] update to 1.14.1

alexkaratarakis committed:
Merge pull request #1795 from KindDragon/update-grpc

Update grpc to version 1.6.0

ras0219-msft committed:
[console-bridge] [urdfdom-headers] Move copyright location. Move CMake config location

ras0219-msft committed:
Merge branch 'master' into Mixaill-urdfdom

ras0219-msft committed:
Merge pull request #1773 from Mixaill/urdfdom

[urdfdom] add version 1.0.0

ras0219-msft committed:
Merge pull request #1779 from OlafvdSpek/master

Update ctemplate, use Python 3

ras0219-msft committed:
Merge pull request #1792 from ShinNoNoir/qt5-removedefaultdeps

[qt5] Do not link all .lib files

alexkaratarakis committed:
[vcpkg_Build.h] Naming scheme, unneeded #include, unneeded inline

codicodi committed:
[xxhash] update to 0.6.3

alexkaratarakis committed:
Merge pull request #1799 from codicodi/xxhash-0.6.3

[xxhash] update to 0.6.3

alexkaratarakis committed:
Merge pull request #1731 from FabienPean/PR/script_cmake_vcpkg_from_bitbucket

[cmake] add function vcpkg_from_bitbucket

KindDragon committed:
[protobuf] Fix define PROTOBUF_USE_DLLS with static linking. Closes #1798

alexkaratarakis committed:
[openssl] Use /Z7 to embed symbols in static libs

alexkaratarakis committed:
Merge pull request #1800 from KindDragon/protobuf-fix

[protobuf] Fix define PROTOBUF_USE_DLLS with static linking

ras0219-msft committed:
[zeromq] Update to latest master to take advantage of CMake improvements.

alexkaratarakis committed:
[openssl] Actually include the patch for /Z7

Commit:
Update pybind11 version to 2.2.0

alexkaratarakis committed:
Update CHANGELOG and bump version to v0.0.87

alexkaratarakis committed:
vcpkg_configure_cmake (and _meson) now embed debug symbols within static libs (/Z7)

alexkaratarakis committed:
Update CHANGELOG and bump version to v0.0.88

KindDragon committed:
[boost] Update to version 1.65.1

alexkaratarakis committed:
Merge pull request #1807 from KindDragon/boost-1.65.1

[boost] Update to version 1.65.1

alexkaratarakis committed:
Merge branch 'master' into pybind11

alexkaratarakis committed:
Merge pull request #1801 from kylemacfarlan/pybind11

[pybind11] Update pybind11 version to 2.2.0

pravic committed:
[sciter] Update to 4.0.3.5348

pravic committed:
[sqlite3] update to 3.20.1

alexkaratarakis committed:
Merge pull request #1809 from pravic/sciter

[sciter] Update to 4.0.3.5348

alexkaratarakis committed:
Merge pull request #1811 from pravic/sqlite

[sqlite3] update to 3.20.1

willyd committed:
[caffe2] Added caffe2 port (CPU only, no OpenCV)

alexkaratarakis committed:
Fix several port hashes due to github tar.gz change

It looks like github changed the version (or the generally the way)
they do tars, causing some hash mismatches. This fixes the affected
ports.

szechyjs committed:
Update to latest python 3.5 patch

alexkaratarakis committed:
Merge pull request #1818 from szechyjs/master

Update to latest python 3.5 patch

alexkaratarakis committed:
Refactor package removal code to eliminate its duplication

alexkaratarakis committed:
[install] Naming scheme and const

alexkaratarakis committed:
Rework vcpkg install (and vcpkg ci)

- Refactor install-plan-execution code to reduce duplication
- Add `vcpkg install --keep-going` option
- Add elapsed time to each invidial package and total time
- Add a counter to the install (e.g. Starting package 3/12: <name>)

glachancecmaisonneuve committed:
Fixed: debug and release versions in their proper folder (was reversed)
Added: Tools included in that library
Misc: portfile cleanup, chm.vcxproj optimization switches

alexkaratarakis committed:
[commands_remove.cpp] Naming scheme and missing const

alexkaratarakis committed:
[vcpkg.cpp] Naming scheme

alexkaratarakis committed:
[metrics.cpp] Naming scheme and missing const

alexkaratarakis committed:
[metrics.cpp] More naming scheme and const fixes

alexkaratarakis committed:
Introduce Version::warn_if_vcpkg_version_mismatch()

alexkaratarakis committed:
Change message of version mismach to a warning

alexkaratarakis committed:
Version check is now performed before running any command

(except vcpkg version/hash/contact)

ras0219-msft committed:
[vcpkg] hotfix for ci/install merge

Disables accidental testing statement that was left in during the commit.

glachancecmaisonneuve committed:
Dynamic build -> Static without failing the build

glachancecmaisonneuve committed:
bump subversion

alexkaratarakis committed:
Remove stray debugging code

alexkaratarakis committed:
`vcpkg edit`: Improve detection for VSCode - Insiders. Prefer it if available

alexkaratarakis committed:
[chmblib] Also modify VCPKG_LIBRARY_LINKAGE

alexkaratarakis committed:
Merge pull request #1822 from glachancecmaisonneuve/chmlib_fix

chmlib fixes, adds and misc

alexkaratarakis committed:
Update CHANGELOG and bump version to v0.0.89

alexkaratarakis committed:
Update CHANGELOG and bump version to v0.0.89

Neumann-A committed:
updatedgoogle benchmark to current trunk due to problems with variadic templated benchmarks with visual studio

Neumann-A committed:
Merge remote-tracking branch 'upstream/master' into benchmark_update

Neumann-A committed:
fix so that the cmake patch is applied correctly

Neumann-A committed:
Removed old hashes

Neumann-A committed:
patch is no longer needed with changed CONFIG_PATH

alexkaratarakis committed:
Bump required version of CMake to 3.9.2

TheCycoONE committed:
Bump luafilesystem to 1.7.0.2

alexkaratarakis committed:
Merge pull request #1827 from TheCycoONE/lfs-17

Bump luafilesystem to 1.7.0.2

nholthaus committed:
Add port for nholthaus/units

nholthaus committed:
git push clean-up of portfile

alexkaratarakis committed:
Merge pull request #1842 from nholthaus/master

new package - units

sjemens committed:
xlnt: update to 1.1.0

ShinNoNoir committed:
[qt5] Update fixcmake.py for new manual-link directory

ShinNoNoir committed:
[qt5] Bump version to reflect fixcmake.py fix

ras0219-msft committed:
Merge pull request #1847 from ShinNoNoir/qt5-fix-cmake-manual-link

[qt5] Update fixcmake.py for new manual-link directories

alexkaratarakis committed:
Merge pull request #1844 from sjemens/xlnt

[xlnt] Update to 1.1.0

alexkaratarakis committed:
[vcpkg edit] Fix VSCode path

alexkaratarakis committed:
Merge pull request #1816 from willyd/caffe2

[caffe2] Added caffe2 port (CPU only, no OpenCV)

ras0219-msft committed:
[paho-mqtt] Remove duplicate libraries without SSL support.

ras0219-msft committed:
[benchmark] Reset to latest stable, pulling back important patch.

ras0219-msft committed:
Merge pull request #1825 from NeumannIMT/benchmark_update

Update google benchmark to current master version

ras0219-msft committed:
Merge pull request #1743 from paercebal/doc_for_tolerance_debug_release_config

Updated the documentation to reflect the greater tolerance on configs

ras0219-msft committed:
Merge pull request #1721 from mmha/kf5plotting

[KF5] Initial port of KPlotting

ras0219-msft committed:
Merge pull request #1699 from UnaNancyOwen/add_winpcap

Add WinPCAP Developer's Pack

ras0219-msft committed:
[cppwinrt] Remove portfile.cmake

ras0219-msft committed:
Merge pull request #1781 from juandent/juandent-patch-1

Juandent patch 1 - portfile for latest version of cppwinrt

ras0219-msft committed:
Merge branch 'apache-arrow-0.6.0' of https://github.com/atkawa7/vcpkg into atkawa7-apache-arrow-0.6.0

ras0219-msft commented:
Thanks for the PR!
